pkg/catalog/catalog_test.go

```diff
@@ -75,10 +75,12 @@ func TestCatalog_ListRepositories(t *testing.T) {
 		{RepositoryID: "repo1", Repository: &graveler.Repository{StorageNamespace: "storage1", CreationDate: now, DefaultBranchID: "main1"}},
 		{RepositoryID: "repo2", Repository: &graveler.Repository{StorageNamespace: "storage2", CreationDate: now, DefaultBranchID: "main2"}},
 		{RepositoryID: "repo3", Repository: &graveler.Repository{StorageNamespace: "storage3", CreationDate: now, DefaultBranchID: "main3"}},
+		{RepositoryID: "repo22", Repository: &graveler.Repository{StorageNamespace: "storage4", CreationDate: now, DefaultBranchID: "main4"}},
```

Comment:
Is there an assumption that they are inserted in order?

Reply:
I don't see how there could be.  However the test _can_ and _should_ verify that they are returned in order, because that's the API.